Automattic/jetpack
# Enhance sharing buttons UI

Fixes https://github.com/Automattic/jetpack/issues/34116 and https://github.com/Automattic/jetpack/issues/34108

## Proposed changes:

After discussion in a primary ticket we are moving forward with the implementation similar to Social Links in Gutenberg.
This PR adds new Sharing Button block alongside Sharing Buttons one that is used as a container. Also, added styles to match existing behaviour.

Note: `Official buttons` style is not yet available and we will add it on later stages. This is also still a dummy block with no logic connected and probably some code to be improved in the future.


https://github.com/Automattic/jetpack/assets/60262784/d8bf1503-acc9-40b8-9b2d-9a0c77634390

### Other information:


## Testing instructions:






- Spin up your site with local changes
- Make sure you have beta block enabled. In your environment (example Docker) wp-config.php you should have `define( 'JETPACK_BLOCKS_VARIATION', 'beta' );`
- Add `Sharing Buttons` block to a post and check it's behaviour.
